demo/main.cpp

```diff
@@ -323,27 +329,28 @@ class BtOgreTestApplication
 		mSDLWindow = SDL_CreateWindow("BtOgre21 Demo", SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, width, height, SDL_WINDOW_SHOWN | SDL_WINDOW_RESIZABLE | (fullscreen ? SDL_WINDOW_FULLSCREEN : 0));
 
 		//Get access to native (os/window manager dependent) window information
-		SDL_SysWMinfo wmInfo;
-		SDL_VERSION(&wmInfo.version);
-		SDL_GetWindowWMInfo(mSDLWindow, &wmInfo);
+		SDL_SysWMinfo* wmInfo = new SDL_SysWMinfo;
```

Comment:
Why do you allocate this structure on the heap? It's useless and I don't see a matching `delete`. Please just kep the wmInof as a struct on the stack

Reply:
It's a misstake, how should i proceed?
Update my branch and do another pull request?

Reply:
Ok i just updated the branch it seems ok to me.